talibs/host/main.c

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#include <err.h>
+#include <tee_client_api.h>
+#include <libs_ta.h>
+
+int main(int argc, char *argv[])
```

Comment:
could be move to xtest regression 1xxx?

Reply:
Well, for regression testing I'd rather refactor the storage TAs. Currently we have two TAs, `storage` and `storage2`, which differ only by their UUID. We could easily move all the common code into a shared library I suppose.